gflownet/envs/base.py

```diff
@@ -33,15 +35,18 @@ def __init__(
         denorm_proxy: bool = False,
         proxy=None,
         oracle=None,
-        proxy_state_format: str = "oracle",
-        fixed_distribution: Optional[dict] = None,
-        random_distribution: Optional[dict] = None,
+        fixed_distribution: dict = None,
+        random_distribution: dict = None,
+        corr_type: str = None,
```

Comment:
Note: to be discussed and possibly updated depending on the decisions from discussions in [PR 108](https://github.com/alexhernandezgarcia/gflownet/pull/108). See also [branch main_misc-ahg](https://github.com/alexhernandezgarcia/gflownet/tree/main_misc-ahg).